Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/server/valueset/VaccinationValueSetApiV1.kt

```diff
@@ -0,0 +1,16 @@
+package de.rki.coronawarnapp.vaccination.core.server.valueset
+
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Header
+import retrofit2.http.Path
+
+interface VaccinationValueSetApiV1 {
+
+    @GET("/version/v1/ehn-dgc/{lang}/value-sets")
+    suspend fun getValueSets(
+        @Header("If-None-Match") etag: String = "",
```

Comment:
Do we need to set that? Could  we just use `okhttp.Cache` which includes handing for this etag/304/if-none-match ?

Reply:
had it first. liked it more this way.
but I reverted.
so:
adjusted :-)